From e2645785353d6cab0eb385539ee855aff0e693c0 Mon Sep 17 00:00:00 2001
From: Prashant S Chauhan <psinghchauha@vmware.com>
Date: Tue, 25 Oct 2022 07:28:27 +0000
Subject: [PATCH] proxy-pom.xml: skip format-code plugin

On released tag tarball 'mvn install' fails to execute format-code
on project proxy because it expects the source to be git repository.
Patch to Skip this with '-DskipFormatCode' while doing mvn install
---
 proxy/pom.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/proxy/pom.xml b/proxy/pom.xml
index f695fcbb..5c658f42 100644
--- a/proxy/pom.xml
+++ b/proxy/pom.xml
@@ -99,6 +99,12 @@
             </goals>
           </execution>
         </executions>
+        <configuration>
+            <skip>${skipFormatCode}</skip>
+            <googleJavaFormatOptions>
+                <aosp>true</aosp>
+            </googleJavaFormatOptions>
+        </configuration>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-- 
2.30.5

